README.md

```diff
@@ -1,7 +1,104 @@
 # Taskcluster Agile Process Tracking
 
+* [Definitions](#definitions)
+* [Tools](#tools)
+* [Roles](#roles)
+* [Sprint process](#sprint-process)
+
+___
+
 ## Definitions
 
+In order to avoid confusion when discussing our process, it's helpful to define the various terms we use to categorize our work at different levels of granularity and how they roll up into each other.
+
+We generally follow [the Atlassian model](https://www.atlassian.com/agile/project-management/epics-stories-themes), with some naming changes to avoid confusion in [our tooling](#Tools).
+
+Theme -> Initiative -> Milestone -> Epic -> Issue
+
+### Theme
+Themes are large focus areas than span the Mozilla organization and are pertinent and addressable by the Taskcluster team.
+
+For Taskcluster @ Mozilla, we are currently tracking the following themes:
+* Making Data Actionable
+* Ease of use
+* Promote external adoption
+* Cost Reduction
+
+### Initiative
+An _initiative_ is a collection of one or more milestones that, taken together, address one or more themes. Depending on the project area, an initiative can be a thin wrapper around a single milestone if that milestone is high value and self-contained.
+
+### Milestone
+The _milestone_ is the goal for a sprint and, as such, should be able to be completed during the sprint duration made up of a small collection of epics. This can be a thin wrapper around a single epic if the user story is large and/or important enough. Milestones can address one or more themes.
+
+### Epic
+An _epic_ a large user story that describes plainly a feature of the Taskcluster system. Epics can address multiple themes. User stories are an important aspect of agile development, but we've chosen to map them to Epics in our usage to make them trackable at the correct level using [our tooling](#Tools).
```

Comment:
Could you add some examples of epics?

Reply:
I'm going to do a second pass to document our current set of Initiatives, but also to add an example of a complete breakdown from Theme to Issue.